scripts/decrypt_config

```diff
@@ -11,6 +11,10 @@ FILE_CLEAR_SERVER=app/server/src/resources/config.json
 
 export GITHUB_ID=$(vault read -field=clientid secret/beebop/auth/github)
 export GITHUB_SECRET=$(vault read -field=secret secret/beebop/auth/github)
+export DOCKER_GITHUB_ID=$(vault read -field=clientid secret/beebop/auth/devdocker/github)
```

Comment:
sorry just wondering what exactly is the `DOCKER_` secrets? is this for local when we want to use docker?